virtuoso-docs-source/xmlsource/rdfperformancetuning.xml

```diff
@@ -897,53 +877,50 @@ CREATE PROCEDURE dump_one_graph
       {
         http (' .\n', ses);
         string_to_file (file_name, ses, -1);
+        gz_compress_file (file_name, file_name||'.gz');
+        file_delete (file_name);
       }
   }
 ;
-</programlisting>
-            </section>
-            <section xml:id="rdfperfdumpandreloadgraphdumponegraphex">
-              <title>Example</title>
-              <orderedlist>
-                <listitem>
-                  <para>Call the </para>
-                  <para>
-                    <emphasis>dump_one_graph</emphasis>
-                  </para>
-                  <para> procedure with appropriate arguments:
-</para>
-                  <programlisting>
+          </programlisting>
+            <para>To load the procedure into Virtuoso, it can simply be copied and pasted into the Virtuoso <code>isql</code> command line tool or Interactive SQL UI of the Conductor and executed.</para>
 
-$ pwd
-/Applications/OpenLink Virtuoso/Virtuoso 6.1/database
+          </section>
+          <section xml:id="rdfperfdumpandreloadgraphsprocexample">
+            <title>Example</title>
 
-$ grep DirsAllowed virtuoso.ini
-DirsAllowed              = ., ../vad, ./dumps
+              <para>Call the <code>dump_one_graph</code> procedure with appropriate arguments:</para>
+
+
+              <programlisting>
+  $ pwd
+  /Applications/OpenLink Virtuoso/Virtuoso 6.1/database
+
+  $ grep DirsAllowed virtuoso.ini
+  DirsAllowed              = ., ../vad,
+
+  $ /opt/virtuoso/bin/isql 1111
+  Connected to OpenLink Virtuoso
+  Driver: 06.01.3127 OpenLink Virtuoso ODBC Driver
+  OpenLink Interactive SQL (Virtuoso), version 0.9849b.
+  Type HELP; for help and EXIT; to exit.
+  SQL> dump_one_graph ('http://daas.openlinksw.com/data#', './data_', 1000000000);
+  Done. -- 1438 msec.
+
+  $
+  As a result, a dump of the graph &lt;http://daas.openlinksw.com/data#&gt; will be found in the files data_XX (located in your Virtuoso db folder):
+
+
+  $ ls
```

Comment:
I question whether `>` instead of `&gt;` is a good change, but to match the rest of these files...
```suggestion

  $
              </programlisting>
              <para>As a result, a dump of the graph <code>&lt;http://daas.openlinksw.com/data></code> will be found in the files <code>data_XX</code> (located in your Virtuoso <code>db</code> folder):
              </para>

              <programlisting>
  $ ls
```